test/sharness/t0160-resolve.sh

```diff
@@ -65,6 +65,8 @@ test_resolve_cmd() {
   test_resolve "/ipfs/$a_hash/b" "/ipfs/$b_hash"
   test_resolve "/ipfs/$a_hash/b/c" "/ipfs/$c_hash"
   test_resolve "/ipfs/$b_hash/c" "/ipfs/$c_hash"
+  test_resolve "/ipld/$dag_hash/a/b/c" "/ipld/$dag_hash/a/b/c"
```

Comment:
Shouldn't that actually resolve to `/iplds/$c_hash`?

Reply:
This is a really confusing test. The DAG object is `{"a": {"b": {"c": "asdfasdf"}}}` (no links). I've changed it to `{"i": {"j": {"k": "asdfasdf"}}}`